ilpycon/templates/homepage_en.html

```diff
@@ -13,11 +13,18 @@
     <section class="jumbotron">
         <div class="container">
             {% include "_messages.html" %}
-            <h1>PyCon Israel 2018</h1>
+            <h1>See you in 2019</h1>
             <p class="lead">
-	      PyCon Israel is happening on June 4-5, 2018! The event will be attended by
-	      hundreds of Pythonistas and sponsored by major international and local companies
-	      that use Python in their day-to-day business.
+	      PyCon Israel 2018 is over.
+	      We've had lot's of fun, and video's will be uploaded to our YouTube channel as soon as we can.
```

Comment:
A couple of style issues in this line, both of course my opinion, feel free to override esp. if you feel very confident in your English:

(1) videos

(2) Don't mix styles: Either "we'll upload vidoes as soon as we can" or "videos will be uploaded as soon as possible"